pkg/services/dashboards/database/database_test.go

```diff
@@ -90,24 +88,6 @@ func TestIntegrationDashboardDataAccess(t *testing.T) {
 		require.False(t, queryResult.IsFolder)
 	})
 
-	t.Run("Should be able to get dashboard by title and folderID", func(t *testing.T) {
```

Comment:
genuine Q - how do you know we can delete this test? do we not need it anymore?

Reply:
it is testing get dashboard by title and `folderID`, we are deprecating folderIDs, we are removing all the folderIDs from the tests, if the test case is testing folderID specific behavior we can remove it

Reply:
I think we need to make sure that we introduce a similar test using `folderUID` instead